### PR TITLE
linux: build lib/common/exceptions

### PR DESCRIPTION
**Pull request by jianchun**
linux: build lib/common/exceptions

Disable SEH usage on Unix. Existing SEH usage can be rewritten or disabled.
Makes cross platform easier.

### CODE REVIEW

**Comment by dilijev** [comment]
@dotnet-bot test this please

File: lib/common/Exceptions/Throw.cpp [hunk 1]
```diff
@@ -122,36 +125,6 @@ namespace Js {
         throw NotImplementedException();
     }
 
-    // Returns true when the process is either TE.exe or TE.processhost.exe
-    bool Throw::IsTEProcess()
```
Comment (dilijev) [inline lib/common/Exceptions/Throw.cpp]: Was this dead code?

**Comment by dilijev** [comment]
LGTM

File: lib/common/Exceptions/Throw.cpp (hunk 1 above)
Comment (jianchun) [inline lib/common/Exceptions/Throw.cpp]: > Was this dead code?

Yes, @curtisman found it.

**Comment by jianchun** [comment]
@dilijev Thanks!

**Pull request merged**